qlib/workflow/task/manage.py

```diff
@@ -69,7 +69,7 @@ class TaskManager:
 
     ENCODE_FIELDS_PREFIX = ["def", "res"]
 
-    def __init__(self, task_pool: str = None):
+    def __init__(self, task_pool: str):
```

Comment:
`task_pool` could be `None` if `TaskManger.list` is called.
Add docs about it in case that other contributors have similar confisions.